### PR TITLE
Add constax package

### PR DESCRIPTION
**Pull request by liberjul**
Add CONSTAX v2, an updated version of the tool described in https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1952-x

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by liberjul** [comment]
@BiocondaBot please add label

File: recipes/constax/build.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+git submodule init AlignmentTools ReadSeq classifier TaxonomyTree > /dev/null 2>&1
+git submodule update > /dev/null 2>&1
+sed -i 's/1.5/1.6/' AlignmentTools/nbproject/project.properties ReadSeq/nbproject/project.properties classifier/nbproject/project.properties
+sed -i 's/basedir="."/basedir="." xmlns:unless="ant:unless"/' classifier/build.xml
+sed -i 's/name="download-traindata" unless="offline"/name="download-traindata" unless="skip_td_download"/' classifier/build.xml
+sed -i 's+move file="${dist.dir}/data.tgz"+move unless:set="skip_td_download" file="${dist.dir}/data.tgz"+' classifier/build.xml
+rm AlignmentTools/parseErrorAnalysis.py
+cd classifier
+ant jar -Dskip_td_download=true > /dev/null 2>&1
+echo "export RDPPATH=$constax/RDPTools/classifier/dist/classifier.jar" >> $constax/pathfile.txt
```
Comment (dpryan79) [inline recipes/constax/build.sh:23]: Could you instead use the rdptools that are already available via conda?

**Comment by liberjul** [comment]
@BiocondaBot please add label

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**